### PR TITLE
iss1 - add bio

### LINKED ISSUE #1: Add a bio

### PR DESCRIPTION
**Pull request by Thomas-Pr**
Resolves: #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (valosekj) [inline README.md]: Please, remove these empty lines.

File: README.md [hunk 2]
```diff
@@ -1 +1,21 @@
-# perrin_project
\ No newline at end of file
+# perrin_project
+
+This is my short bio.
```
Comment (valosekj) [inline README.md]: Please, replace this line with brief info about you. Example [here](https://github.com/brainhack-school2023/bouchard_project/blob/main/README.md).

File: README.md (hunk 1 above)
Comment (Thomas-Pr) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (Thomas-Pr) [reply README.md]: Done!

File: README.md (hunk 1 above)
Comment (valosekj) [reply README.md]: Nice, thanks! 👍🏻

File: README.md (hunk 2 above)
Comment (valosekj) [reply README.md]: Nice, thanks! 👍🏻

**Review by valosekj — APPROVED** [review]
Thanks for your updates! LGTM now!

There are some conflicts with the main branch-- we can check this tomorrow.

**Pull request merged**